packages/migrate/src/types.ts

```diff
@@ -149,12 +150,14 @@ export namespace EngineArgs {
 
   export interface DevDiagnosticInput {
     migrationsList: MigrateTypes.MigrationList
+    schemaFilter: MigrateTypes.SchemaFilter | null
```

Comment:
ℹ️ Yes there is a naming inconsistency here. Already fixing this in https://github.com/prisma/prisma-engines/pull/5533.